SConstruct

```diff
@@ -340,6 +341,17 @@ def scons(): # pylint: disable=too-many-locals
 
     env = Environment(TOOLS=['extra', 'default', 'textfile'])
 
+    # Scons strips out the environment, however to be able to build daos using the interception
+    # library we need to add a few things back in.
+    if 'LD_PRELOAD' in os.environ:
+        # pylint: disable=invalid-sequence-index
+        env['ENV']['LD_PRELOAD'] = os.environ['LD_PRELOAD']
+
+        for key in ['D_LOG_FILE', 'DAOS_AGENT_DRPC_DIR', 'D_LOG_MASK', 'DD_MASK', 'DD_SUBSYS']:
```

Comment:
Minor, we already do similar here:

https://github.com/daos-stack/daos/blob/723ae128931048e6ba9321cdcc7e69e5923a5379/site_scons/prereq_tools/base.py#L609-L616

Should we add these to that list?

Reply:
Also, @phender has requested changes so review needs to be cleared.

Reply:
These are slightly different as you only add them if LD_PRELOAD is set, but they may way to be in the same function at least.  Same goes for the venv env settings I added recently, I'll take a look today and update accordingly.

Reply:
I would be more included to add that code here, and even rename the PreReqComponent() class into something more accurately describing it's current usage.